x-pack/plugins/fleet/server/telemetry/sender.test.ts

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* eslint-disable dot-notation */
```

Comment:
it is there for accessing private properties in unit test, used at many places. it would be ideal not to refer private properties at all in test, though it would take some refactoring